src/renderers/webgl/WebGLPrograms.js

```diff
@@ -177,7 +177,7 @@ function WebGLPrograms( renderer, cubemaps, cubeuvmaps, extensions, capabilities
 			specularIntensityMap: !! material.specularIntensityMap,
 			specularColorMap: !! material.specularColorMap,
 
-			transparent: material.transparent,
+			opaque: ! material.transparent && material.blending !== CustomBlending,
```

Comment:
Nit:

```js
opaque: material.transparent === false && material.blending !== CustomBlending,
```

Reply:
```js
opaque: material.transparent === false && material.blending === NormalBlending,
```

Consider doing that, instead, for now.

Reply:
Done and done!

Reply:
Excellent! That should make everyone happy! 😊